pymove/utils/distances.py

```diff
@@ -2,9 +2,11 @@
 
 
 def haversine(lat1, lon1, lat2, lon2, to_radians=True, earth_radius=6371):
-    """Calculate the great circle distance between two points on the earth (specified in decimal degrees or in radians).
-    All (lat, lon) coordinates must have numeric dtypes and be of equal length.
-    Result in meters. Use 3956 in earth radius for miles
+    """
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Multi-line docstring summary should start at the first line (D212)](https://app.codacy.com/gh/InsightLab/PyMove/pullRequest?prid=5452500)